docs/tasks.md

```diff
@@ -282,6 +284,90 @@ steps:
       sleep 60
     timeout: 5s
 ``` 
+
+#### Specifying `onError` for a `step`
+
+This is an alpha feature. The `enable-api-fields` feature flag [must be set to `"alpha"`](./install.md)
+to specify `onError` for a `step`.
+
+When a `step` in a `task` results in a failure, the rest of the steps in the `task` are skipped and the `taskRun` is
+declared a failure. If you would like to ignore such step errors and continue executing the rest of the steps in
+the task, you can specify `onError` for such a `step`.
+
+`onError` can be set to either `continue` or `fail` as part of the step definition. If `onError` is
+set to `continue`, the entrypoint sets the original failed exit code of the [script](#running-scripts-within-steps)
+in the container terminated state. A `step` with `onError` set to `continue` does not fail the `taskRun` and continues
+executing the rest of the steps in a task.
+
+To ignore a step error, set `onError` to `continue`:
+
+```yaml
+steps:
+  - image: docker.io/library/golang:latest
+    name: ignore-unit-test-failure
+    onError: continue
```

Comment:
Just realizing now that… we could use `onError` with the debug feature too, like `onError: debug` 😅